kyo-data/shared/src/main/scala/kyo/Tag.scala

```diff
@@ -180,119 +180,119 @@ object Tag:
 
     end extension
 
-    /** Base class for type representations in the Tag system. Type provides a structured representation of Scala types that can be
```

Comment:
this is only a refactoring. The earlier versions the new tag impl used to have a more user-friendly `Type` but the final API is quite low-level so I'm moving it to `object internal`